_minutes/2022-03-03-wecg.md

```diff
@@ -0,0 +1,127 @@
+   * [simeon] [Chrome's Off-screen Documents Proposal](https://docs.google.com/document/d/1b-I-vXq2h7OFFmus78jZXIWcKilKJLKLeGplnY9wt7k/edit)
+     * [PR 170](https://github.com/w3c/webextensions/issues/170): Offscreen Documents Proposal
+   * [carlos] WebRequest API
+     * https://github.com/w3c/webextensions/issues/157
+     * https://github.com/w3c/webextensions/issues/164
+     * https://github.com/w3c/webextensions/issues/168
+     * https://github.com/w3c/webextensions/issues/169
+ * **Check-in on ongoing issues**
+
+
+## Attendees (sign yourself in)
+
+ 1. Jack Works (Sujitech)
+ 2. Simeon Vincent (Google)
+ 3. Timothy Hatcher (Apple)
+ 4. Rob Wu (Mozilla)
+ 5. Bradley Cushing (Dashlane)
+ 6. Oliver Dunk (1Password)
+ 7. Krzysztof Modras (Ghostery)
+ 8. Bastien Granger (Dashlane)
+ 9. Giorgio Maone (NoScript)
+ 10. Craig Lurey (Keeper)
+ 11. Carlos Jeurissen (Jeurissen Apps)
+ 12. Tomislav Jovanovic (Mozilla)
+ 13. Frederic Rivain (Dashlane)
+ 14. Todd Schiller (PixieBrix)
+ 15. Philipp Claßen (Ghostery)
+ 16. Dave Herold (Keeper)
+ 17. Zane Bond (Keeper)
+ 18. Alexei (Privacy Badger)
+ 19. Steven McLintock (1Password)
+ 20. Tyler Carson (Keeper)
+ 21. Nir Nahum (WalkMe)
+ 22. Rainer Enders (Keeper Security)
+ 23. Sergey Aldoukhov (Keeper Security)
+ 24. Mukul Purohit (Microsoft)
+ 25. Igor Oleinikov (Grammarly)
+
+
+## Meeting notes
+
+Discussion item: Evidence for performance claims about persistent background scripts
+
+ * [tophf] Produce evidence/statistics on why persistent background scripts are bad for performance
+ * [rob] Tophf isn't here, but I have seen sentiment from developers who are seeking evidence for why Google is pushing for DNR over blocking webRequest.
+ * [simeon] This is not a performance issue, but increasing the privacy guarantees of the platform by moving towards declarative APIs and moving away from APIs that give data to extensions.
+ * [craig] Concerned about this impact given Chrome's deprecation timeline
+ * [krzysztof] MV3 is not improving privacy nor performance, rather harming privacy-protecting extensions
+ * [craig] Examples of detrimental effects on performance (overhead of initializing extensions instead of keeping in memory, e.g. password managers).
+ * [simeon] The storage.session API addresses that class of concerns.
+ * [simeon] I can try to get back to Chrome's extension team to get more concrete evidence / documentation on this.
+ * [alexei] This is a trust issue. Google's actions thus far have not inspired trust.
+ * [jack] MV3 not feature complete, deadline with MV2, not confident that we can resolve issues in time before Chrome fixes its issues.
+ * [craig] Major issue is the deadline. Never seen a transition like this in my 20 years.
+ * [mandoline] What about the vast majority of developers not represented in these meetings? Will their extensions just stop working one day because of Google's poor communication?
```

Comment:
Should "mandoline" here and elsewhere be updated to Frederic?

Reply:
I wrote "mandoline" because that was the Zoom name.

Reply:
@ghostwords I want to merge this PR, and this is the only pending issue. Are you sure that these comments should be attributed to Frederic?

Reply:
I thought that was the person speaking in "mandoline", but I could be wrong!

Reply:
I'd suggest that land the minutes as is and follow up with Dashlane folks to see if we should update "mandoline" in a separate PR.